code-block/README.md

```diff
@@ -18,4 +18,28 @@ yarn dev
 
 If you want to deploy your field plugin to Storyblok, you can do so by using our [CLI](https://www.npmjs.com/package/@storyblok/field-plugin-cli).
 
-
+# App Description
+
+[//]: # (TODO)
+
+## How to set up
+
+In the Block Library, select a block to edit. Add a new field, set the _Field Type_ to _Plugin_, and set the _Custom Type_ to `storyblok-code-block`.
+
+Configure the options:
+
+* `lineStates`: (optional) A JSON array of objects. Each object has two properties: `value` (`string`) and `color` (`string`). For example:
```

Comment:
Reading this, I wonder what if we just exclude the default (neutral) state from the configuration?

I mean users only configure the highlights. And the output include either undefined or null (whatever we choose) for unselected lines. Not-selected lines should be neutral and their color should just follow whatever the background editor color is. So maybe excluding it from the config might look more natural...?

What do you think?

Reply:
Thanks for the feedback! I think I need to clarify this in the readme: by "neutral", I mean that the line is highlighted, but in a neutral color (such as yellow, as opposed to green/red). This would be different from `""`, which means that the line is not highlighted (transparent).

Reply:
How about this example configuration:

<img width="264" alt="image" src="https://github.com/storyblok/field-type-examples/assets/14206504/9e1f4cb9-5b5b-4670-93db-4224e36df6e4">

Reply:
Btw, next week, I am going to document

- the format of the content
- example with highlight.js
- other options for enabling/disabling the title/line offset

Reply:
> @eunjae-lee should we rename this option to `highlightStates`?
> 
> <img alt="image" width="728" src="https://user-images.githubusercontent.com/14206504/239525879-91c853f1-d026-4a4e-ad2e-9e3d0080715b.png">
> May be easier to understand what the option is about.

Oh yeah, it sounds much better 🙌🏻

Reply:
↑ I don't know how this comment is attached here, but anyway, thanks for clarification as well!